documentation/docs/user_guide/inputs/pft_params_nml.md

```diff
@@ -1,5 +1,162 @@
 # pft_params.nml options
 
+In CABLE the characteristics of the vegetation cover are parametrised and generally remain constant throughout the simulation. These **vegetation parameters** in general vary by **vegetation type**.
+CABLE supports tiling of vegetation, i.e. multiple vegetation types with specified area fractions for a single location/grid-cell (also referred to as patches in CABLE). 
+Altogether CABLE considers the 17 distinct types of vegetation cover listed in Table 1.
+Eleven of those are plant functional types (PFT), two are generally unoccupied, and the remaining four are not vegetated. 
+
+## Table 1: CABLE vegetation types
+
+| Number |        Type          |
+|--------|----------------------|
+|    1 	 | Evergreen Needleleaf |
+|    2 	 | Evergreen Broadleaf  |
+|    3 	 | Deciduous Needleleaf |
+|    4 	 | Deciduous Broadleaf  |
+|    5 	 | Shrub 		|
+|    6 	 | C3 Grassland 	|
+|    7 	 | C4 Grassland 	|
+|    8 	 | Tundra 		|
+|    9 	 | C3 Cropland 	        |
+|    10	 |  C4 Cropland 	|
+|    11	 |  Wetland 		|
+|    12	 |  empty 		|
+|    13	 |  empty 		|
+|    14	 |  Barren 		|
+|    15	 |  Urban 		|
+|    16	 |  Lakes 		|
+|    17	 |  Ice 		|
+
+The CABLE vegetation parameters are listed in Table 2 along with their value ranges, units, and descriptions. All parameters in Table 2 are of type real and dimension 17, corresponding to the number of vegetation types in Table 1.
+The ranges listed are “physically possible” and mostly correspond to the optional checks in the CABLE offline code.
+## Table 2: CABLE vegetation parameters
+
+| Name (vegin%)         | Range            | Units                                                                                               | Description |
+|-----------------------|------------------|-----------------------------------------------------------------------------------------------------|-------------|                                                                      
+| 	a1gs            | >=0.0        	   | \( (-) \) 												 | a1 parameter in stomatal conductance model. Represents the sensitivity of stomatal conductance to the assimilation rate |
+| 	alpha           | >=0.0        	   | \( (mol (electrons) \cdot mol^{-1} (photons) (C3) \cdot mol (CO2) \cdot mol^{-1} (photons) (C4)) \) | Initial slope of J-Q response curve |
```

Comment:
Markdown will not split a word in the middle and considers the maths line as 1 word. So I suggest to add a manual break to improve the rendering:
```suggestion
| 	alpha           | >=0.0        	   | \( (mol (electrons) \cdot mol^{-1} (photons) (C3) \cdot \) <br> \( mol (CO2) \cdot mol^{-1} (photons) (C4)) \) | Initial slope of J-Q response curve |
```

Reply:
I am happy with the suggestion to solve the wide column issue. Do you want me to click "Commit suggestion" or you prefer to make the change yourself ?